.gitignore

```diff
@@ -105,4 +105,5 @@ examples/*.tsx
 
 # Files managed by operational-scripts
 tslint.json
+tsconfig.json
```

Comment:
This got auto-generated, happy to remove if not wanted

Reply:
Please remove it… this is due to a bad behaviour of `@operational/script` (a false good idea 😅) cc @TejasQ